exercises/grains/example.rb

```diff
@@ -1,9 +1,14 @@
+module BookKeeping
+  VERSION = 1
+end
+
 module Grains
   def self.square(number)
+    fail ArgumentError if number <= 0 || number > 64
```

Comment:
The adjustment looks reasonable.

BookKeeping is related to the tests, so you might need to add it in so that the example.rb still passes the tests.